package-lock.json

```diff
@@ -10320,6 +10313,7 @@
 				"https://github.com/sponsors/broofa",
 				"https://github.com/sponsors/ctavan"
 			],
+			"license": "MIT",
```

Comment:
Strange that this line was added.

Reply:
I noticed that too; I think I updated `npm`, but I'm not sure what version from, and I can't find anything obvious in the [npm changelog](https://github.com/npm/cli/blob/release/v10/CHANGELOG.md). The [upstream `package.json` license](https://github.com/uuidjs/uuid/blame/v11.0.3/package.json#L21)  hasn't changed in almost a decade.